festim/boundary_conditions/dirichlets/dc_imp.py

```diff
@@ -57,6 +82,9 @@ def create_expression(self, T):
             E_D=self.E_D,
             Kr_0=self.Kr_0,
             E_Kr=self.E_Kr,
+            Kd_0=self.Kd_0,
+            E_Kd=self.E_Kd,
+            P=self.P,
```

Comment:
Should pressure be an Expression object?

Reply:
It should be indeed